lib/menu.rb

```diff
@@ -20,6 +20,6 @@ def view_dishes
   end
 
   def check_if_available(item)
-    return 'That dish is not available. Please select another dish.' if @dishes[item].nil?
+    raise 'That dish is not available. Please select another dish.' if @dishes[item].nil?
```

Comment:
are both of these messages needed? Would just the 'raise' message work without the return one?